### PR TITLE
Guide Logs WIP

### PR DESCRIPTION
**Pull request by soulshake**
## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by soulshake** [comment]
For some reason I can't get "Inspect" to behave as a parent category. The "Inspect" and "Logs" pages both show up as primary categories.

**Comment by soulshake** [comment]
[Guide: Deploy / Logs](https://trello.com/c/WVbtqDQD/1169-guide-deploy-logs)

File: _guide/20-inspect.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: "Inspect"
+permalink: /guide/inspect/
```
Comment (ianmalott) [inline _guide/20-inspect.md]: I think we ended up at calling this section "Monitor." I might have missed a follow-up conversation, though.

File: _guide/21-logs.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Logs"
+permalink: /guide/logs/
+phase: inspect
```
Comment (ianmalott) [inline _guide/21-logs.md]: monitor

**Pull request merged**